app/src/gui/components/notebook/record_table.tsx

```diff
@@ -243,6 +253,14 @@ function RecordsTable(props: RecordsTableProps) {
           minWidth: 150,
           filterable: true,
           renderCell: (params: GridCellParams) => {
+            const {project_id, record_id, revision_id} = params.row;
+
+            const {data: record} = useQuery({
+              queryKey: ['recordData', project_id, record_id, revision_id],
+              queryFn: () =>
+                getFullRecordData(project_id, record_id, revision_id),
+            });
```

Comment:
Surely you can't put a useQuery in a render function - won't you get a bunch of React errors with hooks being used in different orders depending on the contents of the list?